gradle/java-library.gradle

```diff
@@ -28,5 +28,13 @@ test {
     }
 }
 
+tasks.withType(Checkstyle) {
+    reports {
+        xml.enabled false
+        html.enabled true
+        html.stylesheet resources.text.fromFile("$rootDir/config/checkstyle/checkstyle.xsl")
```

Comment:
Yup, what @TimvdLippe said. Actually, he authored this style sheet and asked me to include it in the patch.